fotoapparat/src/main/java/io/fotoapparat/parameter/Zoom.kt

```diff
@@ -0,0 +1,12 @@
+package io.fotoapparat.parameter
+
+/**
+ * Zoom modes which camera can use.
+ */
+sealed class Zoom {
+
+    object FixedZoom : Zoom()
```

Comment:
missing doc e.g. : `The camera can only support one, fixed zoom level`

Reply:
Ok, added.